### ISSUE TITLE
documentation request - coordinates format

### ISSUE THREAD

**Issue by nocturneop15** [toringer/home-assistant-met-next-6-hours-forecast#1]
Great idea to add short term forecast! 

However i've got an issue - this integration is not providing any service or entity. 

cloud you be more specific about what coordinate format to use with long/lat?  I was not able to add coordinates in full format "50.0799811N, 14.3747031E" as the form does not accept letters

**Comment by toringer** [comment]
I reproduced the error for your coordinates. Will look into it later.

Just use the decimals as input, 50.0799811 and 14.3747031.

```
Logger: homeassistant.components.weather
Source: custom_components/met_next_6_hours_forecast/weather.py:184
Integration: Weather (documentation, issues)
First occurred: 17:59:45 (1 occurrences)
Last logged: 17:59:45

met_next_6_hours_forecast: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 493, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 703, in async_device_update
    await task
  File "/config/custom_components/met_next_6_hours_forecast/weather.py", line 184, in async_update
    precipitation_probability=details[
KeyError: 'probability_of_precipitation'

```

**Comment by toringer** [comment]
Fixed in v1.0.3
